torch/_inductor/codegen/cpp.py

```diff
@@ -706,6 +813,10 @@ def logical_and(a, b):
     def logical_or(a, b):
         return f"{a} || {b}"
 
+    @staticmethod
+    def logical_xor(a, b):
```

Comment:
unless you manage to get rid of it, it can go into `common.py` as it's the same between cpu and gpu?

Reply:
https://github.com/pytorch/pytorch/blob/master/torch/_inductor/codegen/common.py#L144